debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -893,6 +965,17 @@ public static AutoCreateMode parse(String value, String defaultValue) {
                     "When 'snapshot.mode' is set as custom, this setting must be set to specify a fully qualified class name to load (via the default class loader)."
                             + "This class must implement the 'Snapshotter' interface and is called on each app boot to determine whether to do a snapshot and how to build queries.");
 
+    public static final Field LOGICAL_DECODING_MESSAGE_HANDLING_MODE = Field.create("logical_decoding_message.handling.mode")
```

Comment:
I like that a lot, especially since it mimics the behavior for the `table.<include|exclude>.list`. That prior art should make it easy to implement in this PR. I'll put that together soon 👍